streams/src/main/java/org/apache/kafka/streams/processor/internals/StateUpdater.java

```diff
@@ -94,6 +94,19 @@ public int hashCode() {
      */
     void remove(final TaskId taskId);
 
+    /**
+     * Pause a task (active or standby) from restoring in the state updater.
+     *
+     * This method does not block until the task is paused.
+     *
+     * The task to be paused is not removed from the restored active tasks and the failed tasks.
```

Comment:
Shouldn't this be:
```
Restored tasks and failed tasks are not paused.
```

or 

```
Tasks in restored active tasks and failed tasks are not paused.
```

Reply:
Could you also add that removed tasks are not paused?

Reply:
ack.